test/src/test/resources/fixtures/scalars/indexed-hydrating-using-json-data-as-arg.yml

```diff
@@ -18,7 +18,7 @@ overallSchema:
     }
   Baz: |
     type Query {
-      baz(data: JSON!): Foo @hidden
+      baz(data: [JSON!]!): [Foo] @hidden
```

Comment:
I think this test had a mistake in the schema definitions, the underlying schema is. Now this test fails because we use the overall field. In practice this schema wouldn't pass semantic validation
```
  Baz: |
    type Query {
      baz(data: [JSON!]!): [Foo]
    }
    type Foo {
      id: ID!
    }
    scalar JSON
```

Reply:
Yea nice catch